challenge1/styles.css

```diff
@@ -15,14 +14,114 @@ html {
     font-family: "Ubuntu", sans-serif;
 }
 
+header {
+    display: flex;
+    justify-content:space-between;
+    align-items: center;
+    padding: 1rem 2rem;
+    gap:2rem
+}
+.filters {
+    display: flex;
+    flex-direction: row;
+    gap: 1rem;
+}
+.title {
+    padding: 0 2rem 0 2rem ;
+}
+
+.search-container {
+    position: relative;
+    display: flex;
+    align-items: center;
+}
+.search-container ::placeholder {
```

Comment:
This isn't super important, but try to be consistent with spacing: sometimes there is an empty line after the brackets and sometimes there isn't